### PR TITLE
[ENG-6252] Git hub tests

### PR DESCRIPTION
**Pull request by Vlad0n20**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -54,6 +54,8 @@ services:
     environment:
       POSTGRES_HOST_AUTH_METHOD: trust
       POSTGRES_DB: gravyvalet
+    ports:
+      - 5435:5432
```
Comment (adlius) [inline docker-compose.yml:58]: Do we have to map the ports?

**Review by adlius — CHANGES_REQUESTED** [review]
Some small questions.

File: addon_imps/tests/_trial_temp/test.log [hunk 2]
```diff
@@ -0,0 +1,2 @@
+2024-10-10 22:56:41+0300 [-] Log opened.
+2024-10-10 22:56:41+0300 [-] /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/unittest/case.py:580: builtins.RuntimeWarning: TestResult has no addDuration method
```
[the remark this answers is not in the document]
Comment (Vlad0n20) [reply addon_imps/tests/_trial_temp/test.log]: Removed these files

File: addon_imps/storage/github.py [hunk 3]
```diff
@@ -84,6 +87,7 @@ async def list_child_items(
                     items=items, total_count=len(items)
                 ).with_cursor(self._create_offset_cursor(len(items), page_cursor))
             elif response.http_status == 404:
+                print("##########  list_child_items 404", await response.json_content())
```
[the remark this answers is not in the document]
Comment (Vlad0n20) [reply addon_imps/storage/github.py]: I forgot to remove test prints. Done

File: addon_service/admin/__init__.py [hunk 4]
```diff
@@ -23,9 +23,6 @@ class ExternalStorageServiceAdmin(GravyvaletModelAdmin):
         "int_addon_imp": known_imps.AddonImpNumbers,
         "int_credentials_format": CredentialsFormats,
         "int_service_type": ServiceTypes,
-    }
-
-    enum_multiple_choice_fields = {
```
[the remark this answers is not in the document]
Comment (Vlad0n20) [reply addon_service/admin/__init__.py]: As I remember it was necessary to admin locally, before you merged fix in develop branch. Returned it

File: docker-compose.yml (hunk 1 above)
Comment (Vlad0n20) [reply docker-compose.yml]: Its not my changes. Removed it

**Pull request merged**